ui/packages/catalog-ui-search/src/main/webapp/component/location-old/inputs/inputs.less

```diff
@@ -0,0 +1,169 @@
+@{reactNamespace}json {
+    display: block;
+    margin-bottom: 10px;
+    position: relative;
+
+    textarea {
+        border-radius: @border-radius;
+        padding: 10px;
+        width: 100%;
+        line-height: 1.2rem;
+        outline: 0;
+        .is-monospaced-font;
+        display: block;
+    }
+
+    .error {
+        padding: 0 10px;
+        .is-negative;
+    }
+
+    button {
+        padding: 0 10px;
+        position: absolute;
+        top: 0;
+        right: 0;
+    }
+}
+
+@{reactNamespace}radio {
+  border-radius: @border-radius;
+  white-space: nowrap;
+  background-color: inherit;
+  display: inline-block;
+  font-size: 0px;
```

Comment:
This is gonna have unintended consequences in IE.  They don't handle reset well unfortunately.